src/explore-education-statistics-admin/src/modules/find-statistics/PublicationReleaseContent.tsx

```diff
@@ -246,7 +250,7 @@ const PublicationReleaseContent = ({
               )}
             </dl>
             <ReleaseNotesSection release={release} />
-            <RelatedInformationSection release={release} />
+            <RelatedInformationSection editing={editing} release={release} />
```

Comment:
We have the EditingContext to get around passing editing state through all components as it begins to get messy.
`const { isEditing } = useContext(EditingContext)` will get you the boolean you're wanting.